impossible.md

```diff
@@ -0,0 +1,23 @@
+IM POS SI BLE
+  - not able to occur, exist, or be done
+  - improbable, utterly impractical
+  - unable to be, exist, happen
+  - not capable of being accomplished
+
+You begin to [ponder](../ponder-about-life.md).
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.